### DISCUSSION TITLE
第4节课作业(16班)

### DISCUSSION THREAD

**Discussion by Yang-Changhui** [InternLM/Tutorial#286] [作业]
16班第四节课作业提交在下面

**Comment by Aitejiu** [comment]
https://www.cnblogs.com/aitejiu/p/17960953

**Comment by Bob-Y** [comment]
### 基础作业：

- 构建数据集，使用 XTuner 微调 InternLM-Chat-7B 模型, 让模型学习到它是你的智能小助手，效果如下图所示，本作业训练出来的模型的输出需要将不要葱姜蒜大佬替换成自己名字或昵称！

微调前：
<img width="960" alt="f73bc9791678b72d36212109589e57b" src="https://github.com/InternLM/tutorial/assets/31715275/cf9b2db9-fa78-4325-9c2e-0b687cb2cbc8">

微调后：
<img width="960" alt="f5cc995e4cebd2103487d03520c3b04" src="https://github.com/InternLM/tutorial/assets/31715275/133baa72-aa1b-471b-9ddd-1eb55e35182c">

**Comment by hhhanqian** [comment]
http://xhslink.com/jhwqkz

**Comment by EnableAsync** [comment]
https://enableasync.github.io/internlm/internlm-04/

**Comment by 2404589803** [comment]
https://2404589803.github.io/My-datawhale-learning/#/shusheng_puyu_learnning53/ch04
![构建数据集，使用 XTuner 微调 InternLM-Chat-7B 模型, 让模型学习到它是你的智能小助手](https://github.com/InternLM/tutorial/assets/68528929/748ce815-a86c-4331-9ca8-5033e14c610e)

**Comment by morcake** [comment]
https://blog.51cto.com/morcake/9234406

**Comment by etheral12138** [comment]
https://y1yc0dwhb0o.feishu.cn/docx/NrxxdbWV6oTeywxojZ3cC6u6nTe?from=from_copylink

**Comment by Durian-1111** [comment]
**Day04-Homework**
链接：https://xby48gs45u.feishu.cn/docx/Cjz5d2QZToZgiyxCJtIcQPCkn5c?from=from_copylink

**Comment by zszbyzsz** [comment]
https://juejin.cn/post/7323135762798952459

**Comment by Eadral** [comment]
https://www.yuque.com/eadral/ruaugi/kdsly7o6vqz4gsh4?singleDoc#

**Comment by Dhaizei** [comment]
![image](https://github.com/InternLM/tutorial/assets/92254983/d1180e2d-c689-4751-9ca3-0c9cf3a26e19)
